docs/cheaha/open_ondemand/index.md

```diff
@@ -25,3 +25,40 @@ For a more complete description of how to select resources, go [here](ood_layout
+
+## Choosing the right Jupyter Application
+
+Jupyter environments have revolutionized the way researchers, analysts, and developers interact with code. Whether you have workflows in Python, visualizing genomic data, running simulations or run any kind of computational workflow on our high-performance computing (HPC) cluster, you've likely encountered using [Jupyter Notebook](./ood_jupyter.md) and/or [JupyterLab](./ood_jupyterlab.md), we have detailed information for using both on Cheaha. Both applications are open-source web tools designed for creating and sharing documents that can include your code, equations, visualizations, and narrative text. Their main purpose is to offer an interactive environment for all phases of your data analysis life cycle including, data exploration, analysis, and visualization. But when should you choose one over the other on Cheaha to run your computational workflow?
+
+This article tries to help you decide, by comparing both Jupyter Apps, looking at their strengths and weaknesses, and how they measure up to your needs, especially on Cheaha.
+
+### What They Have in Common
+
+Before listing out some of the differences, the following are similarities they have in common;
+
+1. They use the same underlying kernel system. (i.e. environments created for use with Jupyter Notebook can also be used in JupyterLab and vice-versa).
+1. They both support `.ipynb` notebooks.
+1. Allow interactive code execution, plotting, markdown, and data visualization.
+1. Can run Python, R, Julia, and other supported languages.
+1. they are both great for reproducing workflows.
+1. You can access both on Cheaha, selecting the configuration and resources required for your workflow.
+
+### Why Choose JupyterLab?
+
+JupyterLab is the next-generation interface for Jupyter. It's designed to be a full-featured web-based IDE. JupyterLab provides an interface for interactive and exploratory computing. Although it includes many features typical of traditional integrated development environments (IDEs), its primary focus remains on interactive and exploratory computing. The following are reasons why you may use JupyterLab over Jupyter Notebook.
+
+1. JupyterLab will be your preferred app, if you are conversant with or your workflow requires you to manage multiple notebooks, terminals, and other files. JupyterLab allows you to in a single tabbed interface, run your code, analyze logs, edit documentation, and move between tasks within your workflow more quickly.
+1. Projects that involve code, markdown, output files, and terminals simultaneously can be split into tabs within a browser tab, without switching browser tabs.
+1. If you want a more customizable and extensible environment, JupyterLab supports extensions out-of-the-box. If you need a Git GUI or a table of contents for navigating long notebooks? JupyterLab lets you add these features easily, making it more like a lightweight IDE than just a notebook viewer.
```

Comment:
There are multiple layers to JupyterLab, so please clarify what part has the extensions. Inexperienced people might think you mean python or Jupyter extensions. I read it as JupyterLab extensions.

It might be helpful to introduce and link to an official extension library if such a thing exists.